### ISSUE TITLE
Nome da ferramenta

### ISSUE THREAD

**Issue by luisfmsouza** [trabalhando/ondetrabalhar#1]
Qual será o nome da nossa ferramenta?

Algumas sugestões:
1. iworkhere
2. myplace
3. justwork
4. bestplace
5. calmhere
6. dohere
7. WorkHere
8. WorkAt...
9. nicespot

ps: Quando decidirmos um nome, podemos ajustar os nomes do repósitorio e organização(aqui no GitHub)

**Comment by klarkc** [comment]
Tem de ver quais domínios estão disponíveis para quais nomes, colocar o domínio `.io` seria uma boa

**Comment by dleitee** [comment]
Gostei de **nicespot**!

**Comment by gabrielmanara** [comment]
Dei uma pesquisada em **workfrom.io** e está disponível!

EDITED
Acabei de ver e já tem projetos com esse nome .co, então nem rola!

**Comment by ortizvinicius** [comment]
trabalha.ai (domínio liberado)

**Comment by luisfmsouza** [comment]
Fiz uma busca no GoDaddy pelas sugestões de nome, e a maioria já está ocupada.

| Nome | Disponível? |
| --- | --- |
| 1. iworkhere(.com) | não |
| 2. myplace(.com) | não |
| 3. justwork(.com) | não |
| 4. bestplace(.com) | não |
| 5. calmhere(.com) | sim |
| 6. dohere(.com) | não |
| 7. WorkHere(.com) | não |
| 8. WorkAt(.com) | não |
| 9. nicespot(.com) | não |
| 10. workfrom(.com) | não |
| 11. trabalha.ai | sim |

Eu gostei do trabalha.ai 👍

**Comment by evansantos** [comment]
trabalha.ai soa legal 💃

**Comment by klarkc** [comment]
gostei do trabalha.ai, o único ponto ruim (ou não, não sei) é que faria sentido apenas em português

**Comment by evansantos** [comment]
Sim :/

**Comment by ortizvinicius** [comment]
Mas o foco não é Brasil mesmo?

**Comment by luisfmsouza** [comment]
A princípio o foco é Brasil sim. :)

**Comment by evansantos** [comment]
mas de certa forma se extende a usuários não brasileiros também, certo? 
Apenas estamos indexando workspaces Brasileiros ao meu ver

**Comment by klarkc** [comment]
pra resolver isso poderíamos ter dois domínios, um para os gringos e um para os brasileiros, sendo que ambos redirecionam para o mesmo servidor, mudando apenas o idioma do site. Podemos começar com trabalha.ai e depois procurar uma outra opção quando formos fazer a versão em inglês

**Comment by luisfmsouza** [comment]
Entendo e concordo com seu ponto @evansantos.

Mas nesse inicio devemos focar em usuários brasileiros, até pra validar toda a proposta da ferramenta e ver se vai atender tudo que buscamos. 

Uma vez que isso foi validado, ai partimos pra traduzir em outras linguas e resolver esses problemas.

**Comment by notnotgabriel** [comment]
trabalha.ai soa bem, voto neste! :grin:

**Comment by joaocarvalhowd** [comment]
trabalha.ai soa bem, voto neste!

**Comment by vicainelli** [comment]
Alguém já comprou o trabalha.ai? Pq parece que [não está mais disponível](https://br.godaddy.com/domains/searchresults.aspx?checkAvail=1&tmskey=&domainToCheck=trabalha.ai).

**Comment by ortizvinicius** [comment]
Uns sites mostram disponível e outros indisponível. Garoteamos?

EDIT: na verdade acho que o goddady não tem suporte a domínios .ai, no marcaria e 101 domains está disponível (não conheço esses sites, só usei para consulta mesmo)

**Comment by vicainelli** [comment]
Realmente, nesses dois está disponível...
Porém, USD 142,00 por 2 anos de registro.. 💸 💸 💸

**Comment by ortizvinicius** [comment]
éééhhh 
Sugiro uma nova issue: patrocínio ou vaquinha?

**Comment by evansantos** [comment]
@vicainelli 142,00?? 

![blackgirlmagic-freddie-harrel-xT0GqyOa60Au5PAQy4](https://camo.githubusercontent.com/bd7a1332bd97606a91210214d4a322763018e6d4/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f7854304771794f61363041753550415179342f67697068792e676966)

**Comment by vicainelli** [comment]
HAHAHAHA YEAP @evansantos 

![screen shot 2016-08-11 at 16 56 53](https://cloud.githubusercontent.com/assets/1155694/17633271/ca58306c-60a2-11e6-9181-5813ad476028.png)

**Comment by ortizvinicius** [comment]
Pior que esses aí são os mais em conta

![captura de tela 2016-08-12 as 15 43 51](https://cloud.githubusercontent.com/assets/1380574/17633417/9710eaea-60a3-11e6-9402-ff5e03935e33.png)

**Comment by ortizvinicius** [comment]
Esess mercenários Anguillanos.

**Comment by vicainelli** [comment]
😳 😳 😳  EUROS?? HAHAHAHAH

Pq não usamos o trabalhai.github.io ?

**Comment by klarkc** [comment]
@vicainelli não vejo problemas na real... Não pra gente, mas pro grande público.... Ter um domínio próprio dá mais seriedade ao projeto do ponto de vista do usuário

**Comment by notnotgabriel** [comment]
Sugiro votarmos em outro, ta custando uma fortuna esse trabalha.ai uhauhauha
iworkhere.com.br, workhere.com.br estão disponíveis e não fogem muito da proposta para o nome.
Fazendo referencia ao jovem nerd, os domínios sotrabalhoaqui.com e eusotrabalhoaqui.com também estão disponíveis auhauhahu

**Comment by ortizvinicius** [comment]
sotrabalhoaqui.com é massa

**Comment by evansantos** [comment]
curti também sotrabalhoaqui.com

**Comment by luisfmsouza** [comment]
Acho que fiquei com uma impressão um pouco errada do sotrabalhoaqui.com. A principio a me passou a impressão de que é pra eleger um único lugar pra trabalhar(que não é a ideia).

Acho que algo na linha de: trabalhedaqui.com.br soaria menos "restritivo".

**Comment by dleitee** [comment]
Raça, eu tava pensando aqui...

Que tal a gente começar o desenvolvimento do MVP e depois pensarmos no nome?

**Comment by notnotgabriel** [comment]
Concordo com a idéia do @luisfmsouza, fica parecendo que vc sempre trabalha em um único lugar.
trabalhedaqui.com.br pode ser uma boa sim :smile:

**Comment by luisfmsouza** [comment]
Bora botar a mão na massa então @dleitee !

Mas essa discussão pode continuar viva.

**Comment by ortizvinicius** [comment]
trabalha.ai sem chances mesmo? :/

Está difícil desapegar kkk

**Comment by ortizvinicius** [comment]
Já que a ideia aqui é brainstorm:

**lugar.io - Liberado e mais barato.**

O que acham?

**Comment by notnotgabriel** [comment]
lugar.io achei muito genérico @ortizvinicius 
@luisfmsouza como vcs estão pensando em fazer pra desenvolver? Vai rolar alguma lista de tasks?

**Comment by luisfmsouza** [comment]
Pensei em criar um trello com todas as funcionalidades que foram listadas na issue #3.

E vamos resolvendo 1 por 1. Ah, pode ser que 1 funcionalidade possa ser quebrada em mais de 1 card.

Mas é só uma ideia. **Poderíamos criar um slack pra discutir a melhor maneira de fazer. O que acham?**

**Comment by evansantos** [comment]
@luisfmsouza waffle.io não é uma boa solução? ele já cria as issues aqui no GH mesmo

**Comment by mateusalexandre** [comment]
Acho uma boa ideia criar um slack @luisfmsouza.

**Comment by klarkc** [comment]
Concordo com o @evansantos O waffle.io vincula as tasks do Github no Kanban, e vice-versa, automaticamente. Dividir a comunidade em duas plataformas vai ser pior para um projeto "pequeno" do ponto de vista dos colaboradores, como este.

**Comment by luisfmsouza** [comment]
Bora lá: https://waffle.io/trabalhando/ondetrabalhar/join

**Comment by luisfmsouza** [comment]
Adicionei o slack no README. 🚀

**Comment by giovannism20** [comment]
E se entrarmos em alguns sites e montarmos vaquinhas, ou montar uma espécie de campanha para conseguir comprar o domínio, afinal isso é para comunidade se soubermos mostrar|vender a ideia para as pessoas ?

**Comment by giovannism20** [comment]
https://www.patreon.com/
https://www.kickstarter.com/

Mas também teríamos que divulgar o projeto criado nas plataformas.
